docs/getting-started.md

```diff
@@ -138,3 +133,41 @@ provide your own certificates, please refer to [Securing Control Plane](/docs/se
 
 To use antctl, the Antrea command-line tool, please refer to this
 [guide](/docs/antctl.md).
+
+## Features
+
+### Antrea Network Policy
+Besides Kubernetes NetworkPolicy, Antrea also implements its own network policy
```

Comment:
```suggestion
Besides Kubernetes NetworkPolicy, Antrea also implements its own NetworkPolicy
```

Reply:
I was trying to differentiate from K8s NetworkPolicy or Antrea NetworkPolicy which are resource/CRD types.
How you think about "Network Policy"?

Reply:
Network Policy sounds good